rest/src/db/CatapultDb.js

```diff
@@ -423,6 +432,57 @@ class CatapultDb {
 			})));
 	}
 
+	/**
```

Comment:
this is quite similar to addBlockMeta (except that it is also adding feeMultiplier). consider refactoring the two functions, if possible.

Reply:
So I had a second look and it wasn't that bad. The new commit shares the `addBlockMetaToEntityList` method.

Reply:
nice job on the refactoring 👍